Tests/Packet++Test/Tests/HttpTests.cpp

```diff
@@ -440,10 +452,13 @@ PTF_TEST_CASE(HttpResponseLayerEditTest)
 
 	PTF_ASSERT_TRUE(responseLayer->getFirstLine()->isComplete());
 	responseLayer->getFirstLine()->setVersion(pcpp::OneDotOne);
+
+	// original status code is 404 Not Found
 	PTF_ASSERT_TRUE(responseLayer->getFirstLine()->setStatusCode(pcpp::HttpResponseStatusCode::Http505HTTPVersionNotSupported));
 	PTF_ASSERT_EQUAL(responseLayer->getFirstLine()->getStatusCode(), pcpp::HttpResponseStatusCode::Http505HTTPVersionNotSupported, enum);
 	PTF_ASSERT_EQUAL(responseLayer->getFirstLine()->getStatusCodeAsInt(), 505);
-	PTF_ASSERT_EQUAL(responseLayer->getFirstLine()->getStatusCodeString(), "HTTP Version Not Supported");
+
+	PTF_ASSERT_EQUAL(responseLayer->getFirstLine()->getStatusCodeString(), "HTTP Version Not Supported"); // customized message from packet
```

Comment:
I'm not sure what the comment means... the message is `"HTTP Version Not Supported"` because we called `setStatusCode(pcpp::HttpResponseStatusCode::Http505HTTPVersionNotSupported)`

Reply:
yeah... sorry, I forgot to do cleanup.

Reply:
The comment is still there...

Reply:
fixed